contrib/olivere/elastic/elastictrace_test.go

```diff
@@ -267,9 +273,10 @@ func checkPUTTrace(assert *assert.Assertions, mt mocktracer.Tracer) {
 	assert.Equal("olivere/elastic", span.Tag(ext.Component))
 	assert.Equal(ext.SpanKindClient, span.Tag(ext.SpanKind))
 	assert.Equal("elasticsearch", span.Tag(ext.DBSystem))
+	assert.Equal(host, span.Tag(ext.NetworkTargetHost))
```

Comment:
I think you could directly set `127.0.0.1` here right? since all the tests use that value.

Reply:
For now, but in the future if that value changes then the tests wont break if we use host